### PR TITLE
BLorient666

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient666.xml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 666</idno>
+                        <altIdentifier>
+                            <idno>Wright cat. CCXXXIII</idno>
+                        </altIdentifier>
+                        <altIdentifier>
+                            <idno>Wright 238</idno>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient666.xml:32]: 233? Or CCXXXVIII?

File: LondonBritishLibrary/orient/BLorient666.xml [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="35 38">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <date notBefore="1667" notAfter="1682"/>
+                                <desc>Written in a fine character by two or three hands, or with several changes of nib.
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient666.xml:107]: maybe, a writing tool? as a more universal term?

**Comment by DariaElagina** [comment]
Thank you!

File: LondonBritishLibrary/orient/BLorient666.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient666.xml:107]: Most probably it is copied by several hands. I see at least one break on fol. 151rb, around the midle of the column. Even though, the hands are not very different.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Comment by thea-m** [comment]
can this be merged?

**Pull request merged**